tests/test_catalog.py

```diff
@@ -457,25 +457,32 @@ def test_estimate_order_from_catalog(order_parameters, requests_mock, auth_mock)
     }
     url_order_estimation = f"{API_HOST}/v2/orders/estimate"
     requests_mock.post(url=url_order_estimation, json=expected_payload)
-    estimation = catalog_instance.estimate_order(order_parameters)
+    estimation = catalog_instance.estimate_order(catalog_order_parameters)
     assert isinstance(estimation, int)
     assert estimation == 100
 
 
-def test_order_from_catalog(order_parameters, order_mock, catalog_mock, requests_mock):
+def test_order_from_catalog(
+    catalog_order_parameters,
+    tasking_order_parameters,
```

Comment:
Why do we need these?

Reply:
You are right, I have added those to test separately and forgot to remove them at the end.